Vg/docs update (#16)
macstadium/packer-plugin-macstadium-orka: docs/builders/config.mdx

* `orka_enable_io_boost` _(bool)_ (optional): Enable [Boost IO Performance](https://orkadocs.macstadium.com/docs/boost-io-performance), off by default  

* `orka_vm_cpu_core` _(int)_ (optional):  Number of cpu cores to use with the virtual machine.  

* `image_precopy` _(bool)_ (optional): The plugin defaults to using image save.  If this option is set to true, the builder will commit the image and perform an image copy. 

# Development / Internal Variables

If you're NOT a dev working on this software you can ignore the following.

But, if you are building/editing/updating this software, you may want to turn on most or all of the following options.  See the [examples/orka.pkr.hcl](./examples/orka.pkr.hcl) or [examples/macos.json](./examples/macos-catalina.json).

* `do_not_delete` _(boolean)_ (optional) _*- for devs*_:  By default this plugin automatically deletes the VM afterwards if all scripts ran successfully. This is useful for debugging builds. 

* `do_not_image` _(boolean)_ (optional) _*- for devs*_ By default this plugin automatically creates an image of the VM after any provisioning steps.  This is useful for debugging builds.   

# Information Notes / Gotchas

> [MacStadium Orka] base images have SSH enabled by default and the username/password is `admin:admin` because they are within' a private network by default.  So this plugin has those credentials hardcoded by default, but you can of course customize the communicator.  See the options from the [SSH Communicator].

Reviewer: ```suggestion
[MacStadium Orka] base images have SSH enabled with username:password  `admin:admin` by default.  See the options from the [SSH Communicator] to see how you can customize that.
```
